README.md

```diff
@@ -163,6 +163,8 @@ Now the result is much better:
 
 ![Grid 3x3](https://github.com/anvaka/VivaGraphJS/raw/master/packages/Images/gridGood.png)
 
+You can tune values during simulation with `layout.simulation.springLength(newValue)`, `layout.simulation.springCoeff(newValue)`, etc.

```

Comment:
I think it should be `simulator` not simulation. You can also add link to simulator settings, which describes all possible values: https://github.com/anvaka/ngraph.physics.simulator/blob/master/index.js#L12